### PR TITLE
button: fix button_view_plain.button_disabled appearance

### LINKED ISSUE #1378: button_theme_islands.button_view_plain.button_disabled: wrong appearance
It looks like normal `button_theme_islands.button_disabled` what is weird.
I would prefer not to fill background in this case.

### PR DESCRIPTION
**Pull request by sipayRT**
Closes #1378

### CODE REVIEW

**Comment by sipayRT** [comment]
@dfilatov 
plain

**Comment by dfilatov** [comment]
lgtm

File: design/common.blocks/button/_theme/button_theme_islands.styl [hunk 1]
```diff
@@ -131,6 +131,11 @@ ctx = '.button_theme_islands'
                 background: #fee481;
             }
         }
+
+        &.button_disabled
+        {
+            background: 0;
```
Comment (tadatuta) [inline design/common.blocks/button/_theme/button_theme_islands.styl]: this actually means `background-position-x: 0; background-*: initial`, I understand it works like expected but looks a bit strange.

Comment (dfilatov) [inline design/common.blocks/button/_theme/button_theme_islands.styl]: let's use `background: none`

Comment (sipayRT) [inline design/common.blocks/button/_theme/button_theme_islands.styl]: @tadatuta here we have to remove background at all. Also, `background-position-x: 0;` – it's default value.

Comment (tadatuta) [inline design/common.blocks/button/_theme/button_theme_islands.styl]: well, I don't care too much but for me `none` looks better

Comment (mishanga) [inline design/common.blocks/button/_theme/button_theme_islands.styl]: +1 for `none`

**Comment by sipayRT** [comment]
@tadatuta changed

**Pull request merged**